docs/tokenFormats/order.md

```diff
@@ -0,0 +1,472 @@
+|                      | `<TokenId>` recipe0: `100`                                        |
+|                      | `<TokenId>` recipe1: `112`                                        |
+|                      | `<TokenId>` recipe2: `163`                                        |
+|                      | `<TokenId>` recipe3: `141`                                        |
+|                      | `<TokenId>` recipe4: `95`                                         |
+|                      | `<TokenId>` recipe5: `156`                                        |
+|                      | `<TokenId>` recipe6: `47`                                         |
+|                      | `<TokenId>` recipe7: `108`                                        |
+|                      | `<TokenId>` recipe8: `34`                                         |
+|                      | `<TokenId>` recipe9: `12`                                         |
+
+### Restrictions
+
+The new `Order` token will have the following restrictions:
+
+```json
+{
+  "SenderHasOutputRole": [
+    {
+      "index": 0,
+      "role_key": "Buyer"
+    }
+  ],
+  "OutputHasRole": [
+    {
+      "index": 0,
+      "role_key": "Supplier"
+    }
+  ],
+  "FixedOutputMetadataValue": [
+    {
+      "index": 0,
+      "metadata_key": "type",
+      "metadata_value": "ORDER"
+    },
+    {
+      "index": 0,
+      "metadata_key": "status",
+      "metadata_value": "submitted"
+    }
+  ],
+  "FixedOutputMetadataValueType": [
+    {
+      "index": 0,
+      "metadata_key": "requiredBy",
+      "metadata_value_type": "Literal"
+    },
+    {
+      "index": 0,
+      "metadata_key": "transactionId",
+      "metadata_value_type": "Literal"
+    }
+  ],
+  "FixedOutputMetadataValueType": [
+    {
+      "index": 0,
+      "metadata_key": "recipe0",
+      "metadata_value_type": "TokenId"
+    },
+    // ... for every recipe in the order
+    {
+      "index": 0,
+      "metadata_key": "recipeN",
+      "metadata_value_type": "TokenId"
+    }
+  ]
+}
+```
+
+For the range of `Recipe` input+output tokens in the order, the following restrictions will apply:
+
+```json
+{
+  "SenderHasInputRole": [
+    {
+      "role_key": "Buyer"
+    }
+  ],
+  "MatchInputOutputRole": [
+    {
+      "input_role_key": "Supplier",
+      "output_role_key": "Supplier"
+    }
+  ],
+  "FixedInputMetadataValue": [
+    {
+      "metadata_key": "type",
+      "metadata_value": "RECIPE"
+    }
+  ],
+  "MatchInputOutputMetadataValue": [
+    {
+      "input_metadata_key": "type",
+      "output_metadata_key": "type"
+    }
+  ]
+}
+```
+
+## POST /order/{id}/rejection
+
+A supplier rejects an order, along with amendment suggestions - a new list of recipes and `requiredBy` date.
+
+| Inputs                    | Outputs                   |
+| :------------------------ | :------------------------ |
+| Order, Recipe0... RecipeN | Order, Recipe0... RecipeN |
```

Comment:
Why do we need to list all the recipes for a rejections?

Reply:
I was thinking a rejection would include a new list of suggested recipes because [NewOrderRejection](https://github.com/digicatapult/inteli-api/blob/d3c7ad9854a3b0d66257838ec19cc6e62a68b3b9/app/api-v1/api-doc.js#L380) includes `items[]`

Thought it would be easiest to send the complete list (of the recipes the supplier wants the order to contain) rather than try to make a token that 'modifies' the original order by adding/removing recipes

Reply:
yes you're right

Reply:
Yes, I think the only way we can do that with our current set of restrictions is by enforcing that every time a `recipe` token is used, its copy has all the same metadata included as in the first token (loads of `MatchInputOutput` restrictions). We maybe don't want to do that if looking to keep tokens small.

Otherwise we could make new restrictions that:
- enforce the abscence of a metadata key on a token
- enforce that only certain keys can be set on a token

Reply:
https://digicatapult.atlassian.net/browse/IN-311